### ISSUE TITLE
Stats per User

### ISSUE THREAD

**Issue by klinger** [coders4help/volunteer_planner#312]
A user should not only see the future shifts she has signed up #300. 
+ She also should have a list of her completed shifts. 
+ And some statistics. At least the total amount of hours of volunteer work. 
+ (wish) A graphical representation like the way github shows the contribution over time in the user pages would be very nice too.
+ These should ONLY be visible to the user.

But where - as an entry in the user menu? "Statistics" or "What I have contributed". The total number of volunteered hours would be nice to be visible the whole time (I will make this suggestion for the user interface draft).

(Wish for future: Some kind of achievements/badges for "your first 20 hours of volunteer work" ).
+ This may be visible to others - some kind of "firsttimer/bronze/silver/platin" icons near to the name of the user in the in shift overview list. This would be a rough estimate of the experience of the users, this may increase the user experience and give the shift planners additional information.

**Comment by Doca** [comment]
Yes badges will come later i think. Some basic stats will come also with the ui rework.

**Comment by klinger** [comment]
+ "She also should have a list of her completed shifts." (fixed by #346 )
